frontend/src/views/Pages/UserProfile/UserCard_CoinAnimation.spec.js

```diff
@@ -45,5 +47,84 @@ describe('UserCard_CoinAnimation', () => {
     it('has an edit BFormCheckbox switch', () => {
       expect(wrapper.find('.Test-BFormCheckbox').exists()).toBeTruthy()
     })
+
+    describe('enable with success', () => {
+      beforeEach(async () => {
+        await wrapper.setData({ CoinAnimationStatus: false })
+        mockAPIcall.mockResolvedValue({
+          data: {
+            updateUserInfos: {
+              validValues: 1,
+            },
+          },
+        })
+        await wrapper.find('input').trigger('change')
+      })
+
+      it('calls the updateUserInfos mutation', () => {
+        expect(mockAPIcall).toBeCalledWith({
+          mutation: updateUserInfos,
+          variables: {
+            coinanimation: false,
```

Comment:
I would expect true here, At the beginning `CoinAnimationStaus = false`. When I trigger change, I expect the value to be `true`

Reply:
Not sure about this but I think you are right. I changed it, we see what happens.

Reply:
resolving this since fixed and relevant conversation is below